### PR TITLE
ASCII Table: Add additional trigger words

### LINKED ISSUE #4107: ASCII Codes Table: Add additional trigger words
# Background

The current existing hard-coded iframe appears for the following queries:

- ascii table
- ascii codes
- char codes
- character codes
- ascii character codes
- ascii codes
- ascii characters

## What to do

- Add the above trigger words to the `triggers` in `[HtmlEntities.pm](https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/lib/DDG/Goodie/HtmlEntities.pm#L13)`
- Adjust the regex in the `handle` function accordingly

------
IA Page:  http://duck.co/ia/view/ascii_table 
[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @ManrajGrover

### PR DESCRIPTION
**Pull request by adityatandon007**
## Related Issues and Discussions

Fixes #4107 

## People to notify
@moollaza @pjhampton 

## Testing & Review
To be completed by Community Leader (or DDG Staff) when reviewing Pull Request

**Pull Request**
- [x] Title follows correct format (Specifies Instant Answer + Purpose)
- [x] Description contains a valid Instant Answer Page Link (e.g. https://duck.co/ia/view/my_ia)

**Code**
- [x] Adheres to the [DuckDuckGo Style Guide](https://docs.duckduckhack.com/resources/code-style-guide.html)
- [ ] Behaviour is appropriately tested. If improvement, tests are adequately extended.
- [x] There is no unnecessary files in place (such as editor config files)
- [x] There is no API keys / secrets present
- [x] Tests are passing (run `$ duckpan test <goodie_id>`)
    - Tester should report any failures

**Ready to merge?**

- [ ] Has this IA been deployed to and tested on [beta.duckduckgo.com](https://beta.duckduckgo.com/)?
- [ ] For larger commits, has this been approved be more than one community member?
- [ ] The number of reviews is appropriate for this type of PR
- [ ] The commit message is clear, coherent and fitting

**Pull Request Review Guidelines**: https://docs.duckduckhack.com/programming-mission/pr-review.html

---

Instant Answer Page: https://duck.co/ia/view/ascii_table

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: lib/DDG/Goodie/AsciiTable.pm [hunk 1]
```diff
@@ -11,7 +11,9 @@ zci answer_type => 'ascii_table';
 zci is_cached => 1;
 
 triggers start => 'ascii table', 'ascii tables', 'ascii reference table',
-                  'ascii reference';
+                  'ascii reference', 'ascii codes', 'char codes', 'character codes',
```
Comment (ManrajGrover) [inline lib/DDG/Goodie/AsciiTable.pm]: @adityatandon007 Let's move the triggers to separate file and read the file here.

**Comment by adityatandon007** [comment]
@moollaza @ManrajGrover changes done as requested

**Comment by moollaza** [comment]
@adityatandon007 Thanks! This LGTM :+1:

@adityatandon007 would you like to also add support for queries like 'list of ascii characters'?

You'll want to add change the `triggers` to `startend` and in the `handle` strip out the extra words:

```perl
handle query_lc => sub {
    s/^list of\b//;
    return unless $_ == '';
    ...
```

**Comment by moollaza** [comment]
@adityatandon007 looking at our autocomplete, it looks like there's also a few more terms that we could add:

- ascii character(s) list
- ascii chacters lookup
- ascii character map
- ascii character set table

**Comment by ManrajGrover** [comment]
@moollaza I can take fix that

**Comment by pjhampton** [comment]
Awesome! Thanks @ManrajGrover! <3

**Pull request merged**